composer.json

```diff
@@ -8,6 +8,12 @@
         "chat": "https://www.drupal.org/node/314178",
         "docs": "https://www.drupal.org/docs/user_guide/en/index.html"
     },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
```

Comment:
This makes `composer.json` not normalized; I think we need to add `--dry-run` in https://github.com/itk-dev/ai-screening/blob/feature/hooks/.github/workflows/pr.yml#L48:

```
          docker compose run --user root --rm phpfpm composer normalize --dry-run
```